### ISSUE TITLE
Selected text in the textarea is selected in VS Code, too, but not quite

### ISSUE THREAD

**Issue by VOOM108** [fregante/GhostText#326]
### Description and repro

It is nice that selected text in the textarea will be selected in VS Code, too. But it seems to be a few lines off. Also, it would be nice, if the selected portion of the text would be instantly in the viewport in VS Code, if that is possible at all. 

### Browser(s) tried

Arc, Chrome

### Editor(s) tried

VS Code

**Comment by fregante** [comment]
Can you provide a video? The selection works for me

**Comment by VOOM108** [comment]
This is the typical offset I get:

![Zwischenablagebild (6)](https://github.com/user-attachments/assets/c8df0a0e-f35b-4933-b680-8af2554432d1)

![Zwischenablagebild (7)](https://github.com/user-attachments/assets/2bd845d2-025f-44bc-811c-a212b9a5e27e)

**Comment by fregante** [comment]
Are you using grammarly?

Please test it on ghosttext.fregante.com/troubleshooting

**Comment by VOOM108** [comment]
No, that is Language Tool. I turned it off for the test with the troubleshooting-page. The selection has an offset to the top there, too, though less.

**Comment by fregante** [comment]
So, a couple of notes:

- **browser** to **editor** selection is not updated, there's no `selectionchange` listener
- **editor** to **browser** selection is implemented exclusively for raw `textarea` elements, and only for one selection (not multiple selections)
- **editor** to **browser textarea** selection works correctly in my tests (VS Code -> Chrome)

I'd say:

- make sure you're using the official VS Code extension: https://marketplace.visualstudio.com/items?itemName=fregante.ghost-text
- try with another editor (e.g. Sublime), it's possible that the selection index is being changed by some other editor plugin you have installed
